neuralprophet/time_net.py

```diff
@@ -880,7 +880,7 @@ def all_covariates(self, covariates):
                 x = x + self.covariate(lags=covariates[name], name=name)
         return x
 
-    def forward(self, inputs, meta=None):
+    def forward(self, inputs: Dict, meta: Dict = None) -> torch.tensor:
```

Comment:
```suggestion
    def forward(self, inputs: Dict, meta: Dict = None) -> torch.Tensor:
```

I received several warnings about `torch.tensor` being a function instead of a type. Maybe we better use `torch.Tensor` (upper case letter referring to the class instead of the function).

Reply:
sg